src/test/java/domain/GameTest.java

```diff
@@ -0,0 +1,55 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class GameTest {
+
+    private Game game;
+
+    @BeforeEach
+    void beforeEach() {
+        List<PlayerName> usernames = List.of(new PlayerName("a"), new PlayerName("b"), new PlayerName("c"));
```

Comment:
```suggestion
        List<PlayerName> usernames = Stream.of("a", "b", "c").map(PlayerName::new).toList();
```

개인 취향이지만, 저는 생성자 호출을 반복 사용하는 것을 선호하지 않아서 stream 처리를 애용해요. 객체 생성에 주어지는 given 값이 중요한데, 생성자로 감싸지다보니 시선이 흐려지더라고요.